taskcat/lambda_build.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+# authors:
+# Tony Vattathil <tonynv@amazon.com>, <avattathil@gmail.com>
+# Santiago Cardenas <sancard@amazon.com>, <santiago[dot]cardenas[at]outlook[dot]com>
+# Shivansh Singh <sshvans@amazon.com>,
+# Jay McConnell <jmmccon@amazon.com>,
+# Andrew Glenn <andglenn@amazon.com>
+from __future__ import print_function
+
+
+import os
+from shutil import make_archive
+from taskcat.colored_console import PrintMsg
+
+
+class LambdaBuild(object):
```

Comment:
I think we can stop using  (object) param in class definitions. As of Python3 this is no longer required.

Reply:
Yeah, I agree it seems redundant. my IDE still suggests I do it in p3, probably pep20 `Explicit is better than implicit.` mantra.

Reply:
which ide and package do you use @jaymccon? :) I use sublime and it doesn't complain that...lol #INeedBetterIDE

Reply:
pycharm